src/main/scala/com/yotpo/metorikku/configuration/job/Periodic.scala

```diff
@@ -0,0 +1,15 @@
+package com.yotpo.metorikku.configuration.job
+
+import com.yotpo.metorikku.exceptions.MetorikkuException
+
+import scala.concurrent.duration.Duration
+
+case class Periodic(triggerDuration: Option[String]) {
```

Comment:
you would be able to test this, if you can also add numberOfRuns here somehow, otherwise, you can't really test the feature